traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -159,18 +159,12 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodGet, `user/{id}/deliveryservices/available/?(\.json)?$`, user.GetAvailableDSes, auth.PrivLevelReadOnly, Authenticated, nil},
 		{1.1, http.MethodPost, `user/login/?$`, login.LoginHandler(d.DB, d.Config), 0, NoAuth, nil},
 
-		//User: CRUD (1.4 then 1.1)
-		//Incrementing version for users because change to Nullable struct. Delete is completely new.
+		//User: CRUD
+		//Incrementing version for users because change to Nullable struct.
 		{1.4, http.MethodGet, `users/?(\.json)?$`, api.ReadHandler(user.GetTypeSingleton()), auth.PrivLevelReadOnly, Authenticated, nil},
```

Comment:
Thanks for deleting the explicit Perl 1.1 routes, but I think the go routes need to be 1.1 now too so that the Go handler will serve 1.1 and above, not just 1.4.